WordPress/gutenberg
# Theme JSON schema: add heading/button key to color definition

Follow-up on #49131
Similar to #55675

## What?

This PR adds heading/button key to color definition in theme.json schema.

## Why?
To allow developers to design theme.json with reference to the correct schema.

## Testing Instructions

Create the following JSON file locally, specifying the JSON schema changed by this PR.

```json
{
	"$schema": "https://raw.githubusercontent.com/WordPress/gutenberg/schema/theme-button-heading/schemas/json/theme.json",
	"version": 2
}
```

- You should see the `button` and `heading` keys listed in the `settings.color` property.
- You should see the `button` and `heading` keys listed in the `settings.block.{blockName}.color` property.
- The default value of `true` should be set when the `button` or `heading` key is selected.

![capture](https://github.com/WordPress/gutenberg/assets/54422211/872adcd6-2dd3-49eb-81b4-e8cc8afd88c7)
